crc-e2e/task.yaml

```
@@ -0,0 +1,325 @@
+          platform: XXXX
+          os-version: XXXX
+          arch: XXXX
+          os: XXXX
+    - name: secret-quay
+      default: registry-quay-io-crcont-qeplatform
+    # crc params
+    - name: crc-version
+      description: crc version to be tested (i.e 2.34.1)
+      # default: latest
+    - name: bundle-location
+      description: custom bundle already downloaded on the target host. 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)